### PR TITLE
Removing docs that were added to website

### PR DESCRIPTION
**Pull request by mikewalch**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,128 +1,12 @@
...
-```
+* [Documentation] for running a Fluo application in Docker
 
 [Fluo]: https://fluo.apache.org/
-[Accumulo]: https://accumulo.apache.org/
-[Hadoop]: https://hadoop.apache.org/
-[Zookeeper]: https://zookeeper.apache.org/
-[application]: https://github.com/apache/fluo/blob/master/docs/applications.md
-[Marathon]: https://mesosphere.github.io/marathon/
+[Documentation]: https://fluo.apache.org/docs/fluo/1.2/administration/run-fluo-in-docker/
```
Comment (keith-turner) [inline README.md]: Is it possible to have  a link w/o a version?

**Review by keith-turner — APPROVED** [review]
on related website PR moving build docs back to readme was discussed.

File: README.md [hunk 2]
```diff
@@ -8,5 +8,56 @@ This project creates the official [Apache Fluo][Fluo] docker image.
...
+|-------------|---------|
+| [Fluo]      | 1.2.0   |
+| [Accumulo]  | 1.8.1   |
+| [Hadoop]    | 2.7.3   |
+| [Zookeeper] | 3.4.9   |
+
+If these versions do not match what is running on your cluster, you should consider building
+your own image with matching versions. However, Fluo must be 1.2+.
+
+## Build the Docker image
```
Comment (keith-turner) [inline README.md]: could add a comment that this section name is linked from web docs and needs to be stable....

File: README.md (hunk 1 above)
Comment (mikewalch) [reply README.md]: It is but we would probably need to update that link with each release.

**Pull request merged**